callback.php

```diff
@@ -108,6 +103,19 @@
     echo 'Secret not matching';
 }
 
+function insertTXIDIntoPaymentDetails($presta_order, $txid, $blockonomics_order)
+{
+    $btc_price = $blockonomics_order['bits']/$blockonomics_order['value'];
+    $amount = $blockonomics_order['bits_payed'] / $btc_price;
```

Comment:
Calculation of the amount is working correctly when using different currencies. We can restructure this to be more similar to the WHMCS module which instead uses the paid to expected ratio to calculate the amount to credit.
https://github.com/blockonomics/whmcs-bitcoin-plugin/blob/master/modules/gateways/callback/blockonomics.php#L84-L85
```
$paid_ratio = $blockonomics_order['bits_payed'] / $blockonomics_order['bits'];
$amount = $paid_ratio * $blockonomics_order['value'];
```

Reply:
Nice, I've updated it to reflect the WHMCS module. Thanks for the tip!